src/test/scala/dotty/semanticdb/Semanticdbs.scala

```diff
@@ -18,7 +18,7 @@ object Semanticdbs {
    */
   class Loader(sourceroot: Path, classpath: List[Path]) {
     private val META_INF = Paths.get("META-INF", "semanticdb").resolve(buildSubFolder)
-    private val classLoader = new java.net.URLClassLoader(classpath.map(_.toUri.toURL).toArray)
+    private val classLoader = new java.net.URLClassLoader(classpath.map(_.toUri.toURL).toArray[java.net.URL | JavaNull])
```

Comment:
Something that would be fixed if method required argument of type A but requires argument of type A|Null
(typical example: pass foo.toArray as argument to a Java method that requires an array)
This could be fixed by better Dotty type inference when prototype involves union.